### DISCUSSION TITLE
Add option to track multi-train journeys automatically and also substitutional services (when connections are missed), link to original trip

### DISCUSSION THREAD

**Discussion by b-pfl** [Traewelling/traewelling#2048] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
- Especially when traewelling using the last connection(s) of a day, it can happen that somewhere on the journey, the user misses the last connecting train(s) and needs to take a substitutional service (e.g. a taxi paid by DB).
- On longer multi-train/-bus connections, I regularly forget to check in on subsequent trains. 

**Describe the solution you'd like**
- To improve usability for multi-train journeys, it would be great to enable a check-in for the whole journey at the beginning of a ride (with having an option to adjust the journey during/after the ride in case of delays and re-routings). 

- In order to be able to use traewelling as a source to check the eligibility for compensations (Verspätungsentschädigung), it would be great to be able to  enter the complete (intended) connection (see item above), and then allow for 
a) defining segments of the journey in the form of: "I used a different mode of transportation (e.g. taxi)", enter actual arrival time

**Describe alternatives you've considered**

**Additional context**

**Comment by MrKrisKrisu** [comment]
could be a duplicate of #769
